src/utils/MySQLPoolDataHandler.py

```diff
@@ -1161,6 +1248,35 @@ def UpdateJobTextField(self, jobId, field, value):
                 conn.close()
         return ret
 
+    @record
+    def UpdateJobTextFields(self, jobId, fields):
+        conn = None
+        cursor = None
+        ret = False
+        if not isinstance(fields, dict) or not fields:
+            return ret
+
+        try:
+            sql = "update `%s` set" % (self.jobtablename)
+            for field, value in fields.items():
```

Comment:
A one liner: `",".join([" `%s` = '%s'" % (field, value) for field, value in fields.items()])`